Polygon_mesh_processing/test/Polygon_mesh_processing/test_shape_predicates.cpp

```diff
@@ -207,9 +208,9 @@ void test_needles_and_caps(const char* fname)
 
   res = PMP::is_cap_triangle_face(f, mesh, 0./*cos(pi/2)*/);
   assert(mesh.point(target(res, mesh)) == Point_3(0,0,1));
-  res = PMP::is_cap_triangle_face(f, mesh, std::cos(boost::math::constants::two_thirds_pi<FT>()));
+  res = PMP::is_cap_triangle_face(f, mesh, std::cos(2 * CGAL_PI / 3));
```

Comment:
`std::cos(2 * CGAL_PI / 3)` is also a constant, equal to 0.5... :smile:

Reply:
More like -0.5 :)

The angle is just clearer.

Reply:
I had not seen that it was in a test .cpp file. No need to over-optimize.